arch/arm/soc/atmel_sam/sam4s/Kconfig.soc

```diff
@@ -4,6 +4,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
+# The default settings in this file were for the board SAM4S Xplained.
```

Comment:
Actually we configure the SoC here to operate at its maximum supported values, i.e. normal operation. It's not really SAM4S Xplained board specific.